README.md

```diff
@@ -35,10 +38,12 @@ For example:
 	}
 }
 ```
+The value `opentable` is referenced via `$(foo.bar)` in your response body.
+
 ```
-<foo><bar>opentable</bar></foo>
+<root><foo><bar type="string">opentable</bar></foo></root>
```

Comment:
No. The root element doesn't present in the resulting map.
For case: 
```
Map object = xmlMapper.readValue("<request type=\"object\"><getTaxRequest type=\"object\"><entry0 type=\"object\"><country type=\"string\">US</country></entry0></getTaxRequest></request>", Map.class);
```
Map will have 2 entries: type -> object and getTaxRequest -> List size = 2 ().
That's why the root element must be omitted.